dvc/remote/gdrive.py

```diff
@@ -83,7 +93,8 @@ def init_drive(self):
             else self.config.get(
```

Comment:
in the line above with `tmp_fname(".dvc/tmp/")`, should it actually append the repo path?

Reply:
cc @efiop  ^^

Reply:
@shcheklein Not sure what you mean, could you clarify? What I there is that it will result in `/path/.dvc/tmp.UUID.tmp`, which is probably not what you want.

Reply:
Makes sense, have changed above line to `tmp_fname(self.repo.tmp_dir)`

Reply:
@MaxRis To `tmp_fname(self.repo.tmp_dir)` ? It will have the same issue I've pointed to above.

Reply:
@efiop you're right, thanks
Have changed it to `tmp_fname(os.path.join(self.repo.tmp_dir, ""))`
Execution results to something like: `/home/max/test/dvc-test/test12/.dvc/tmp/.7wgeWSLfKSbLsDngLPn2RZ.tmp`

I guess, @shcheklein , meant that the path should be absolute and relative to repo's root.